addBagFunction.php

```diff
@@ -0,0 +1,11 @@
+<?php
+function addBag () {
+    $brand = $_POST['brand'];
+    $model = $_POST['model'];
+    $price = intval($_POST['price']);
+    $materials = $_POST['materials'];
+    $image = $_POST['image'];
+    $query=$db->prepare ("INSERT INTO `Bags` (`brand`, `image`, `model`, `price`, `materials`)VALUES(?, ?, ?, ?, ?);");
+    $query -> execute([$brand, $image, $model, $price, $materials]);
```

Comment:
I know you're using this method of binding parameters on Rich's advice, but bear in mind that it's less clear which values you're inserting to which field. Ash might ask you to use $query->bindParam() instead.